views/components/form-fields-next/macros.njk

```diff
@@ -113,14 +113,22 @@
 {% endmacro %}
 
 {% macro inputDateRange(field) %}
-    <fieldset class="ff-date ff-date--range" id="field-{{ field.name }}">
+    <fieldset class="ff-date ff-date--range" id="field-{{ field.name }}-legend">
```

Comment:
Not quite sure about this markup, a fieldset with only a legend in it is a bit wonky. You can _nest_ fieldsets though. Not 100% what the right markup is here, maybe this top one should be a heading and then the two fields are fieldsets.

Reply:
Tried it as a div/header combo, have a look now!

Reply:
👍 Makes sense to me